launch-or-update-local.ps1

```diff
@@ -32,7 +32,37 @@ param(
         [switch]$u=$true
     )
 
+# Parameters   
 $chart="src/buyingcatalogue"
+$namespace="buyingcatalogue"
+
+# Check context is docker dashboard
+$context = kubectl config current-context
+if ($context -ne "docker-desktop")
+{
+    write-host "Not running in the local context - please switch to docker desktop"
+    write-host "Exiting...."
+    start-sleep 20
+    exit 1
+}
+
+# Add namespace if missing
+$namespacePresent = kubectl get namespace $namespace
+if (!($namespacePresent))
+{
+    kubectl create namespace $namespace
+    write-host "`nNamespace has been created`n"
+}
```

Comment:
Would we want to do this in the same way that the selenium grid does it? (i.e. `kubectl apply -f local-namespace.yml`)

Reply:
Not really much difference I think, but no problem - I'll amend the script to run that